// @progress/kendo-react-data-tools/dist/es/messages/index.js
var _a;
/**
 * @hidden
 */
export var pagerInfo = 'pager.info';
/**
 * @hidden
 */
export var pagerFirstPage = 'pager.firstPage';
/**
 * @hidden
 */
export var pagerPreviousPage = 'pager.previousPage';
/**
 * @hidden
 */
export var pagerNextPage = 'pager.nextPage';
/**
 * @hidden
 */
export var pagerLastPage = 'pager.lastPage';
/**
 * @hidden
 */
export var pagerItemPerPage = 'pager.itemsPerPage';
/**
 * @hidden
 */
export var pagerPage = 'pager.page';
/**
 * @hidden
 */
export var pagerOf = 'pager.of';
/**
 * @hidden
 */
export var pagerTotalPages = 'pager.totalPages';
/**
 * @hidden
 */
export var filterEqOperator = 'filter.eqOperator';
/**
 * @hidden
 */
export var filterNotEqOperator = 'filter.notEqOperator';
/**
 * @hidden
 */
export var filterIsNullOperator = 'filter.isNullOperator';
/**
 * @hidden
 */
export var filterIsNotNullOperator = 'filter.isNotNullOperator';
/**
 * @hidden
 */
export var filterIsEmptyOperator = 'filter.isEmptyOperator';
/**
 * @hidden
 */
export var filterIsNotEmptyOperator = 'filter.isNotEmptyOperator';
/**
 * @hidden
 */
export var filterStartsWithOperator = 'filter.startsWithOperator';
/**
 * @hidden
 */
export var filterContainsOperator = 'filter.containsOperator';
/**
 * @hidden
 */
export var filterNotContainsOperator = 'filter.notContainsOperator';
/**
 * @hidden
 */
export var filterEndsWithOperator = 'filter.endsWithOperator';
/**
 * @hidden
 */
export var filterGteOperator = 'filter.gteOperator';
/**
 * @hidden
 */
export var filterGtOperator = 'filter.gtOperator';
/**
 * @hidden
 */
export var filterLteOperator = 'filter.lteOperator';
/**
 * @hidden
 */
export var filterLtOperator = 'filter.ltOperator';
/**
 * @hidden
 */
export var filterIsTrue = 'filter.isTrue';
/**
 * @hidden
 */
export var filterIsFalse = 'filter.isFalse';
/**
 * @hidden
 */
export var filterAfterOrEqualOperator = 'filter.afterOrEqualOperator';
/**
 * @hidden
 */
export var filterAfterOperator = 'filter.afterOperator';
/**
 * @hidden
 */
export var filterBeforeOperator = 'filter.beforeOperator';
/**
 * @hidden
 */
export var filterBeforeOrEqualOperator = 'filter.beforeOrEqualOperator';
/**
 * @hidden
 */
export var filterAndLogic = 'filter.andLogic';
/**
 * @hidden
 */
export var filterOrLogic = 'filter.orLogic';
/**
 * @hidden
 */
export var filterAddExpression = 'filter.addExpression';
/**
 * @hidden
 */
export var filterAddGroup = 'filter.addGroup';
/**
 * @hidden
 */
export var filterClose = 'filter.close';
/**
 * @hidden
 */
export var messages = (_a = {},
    _a[pagerItemPerPage] = 'items per page',
    _a[pagerInfo] = '{0} - {1} of {2} items',
    _a[pagerFirstPage] = 'Go to the first page',
    _a[pagerPreviousPage] = 'Go to the previous page',
    _a[pagerNextPage] = 'Go to the next page',
    _a[pagerLastPage] = 'Go to the last page',
    _a[pagerPage] = 'Page',
    _a[pagerOf] = 'Of',
    _a[pagerTotalPages] = '{0}',
    _a[filterEqOperator] = 'Is equal to',
    _a[filterNotEqOperator] = 'Is not equal to',
    _a[filterIsNullOperator] = 'Is null',
    _a[filterIsNotNullOperator] = 'Is not null',
    _a[filterIsEmptyOperator] = 'Is empty',
    _a[filterIsNotEmptyOperator] = 'Is not empty',
    _a[filterStartsWithOperator] = 'Starts with',
    _a[filterContainsOperator] = 'Contains',
    _a[filterNotContainsOperator] = 'Does not contain',
    _a[filterEndsWithOperator] = 'Ends with',
    _a[filterGteOperator] = 'Is greater than or equal to',
    _a[filterGtOperator] = 'Is greater than',
    _a[filterLteOperator] = 'Is less than or equal to',
    _a[filterLtOperator] = 'Is less than',
    _a[filterIsTrue] = 'Is true',
    _a[filterIsFalse] = 'Is false',
    _a[filterAfterOrEqualOperator] = 'Is after or equal to',
    _a[filterAfterOperator] = 'Is after',
    _a[filterBeforeOperator] = 'Is before',
    _a[filterBeforeOrEqualOperator] = 'Is before or equal to',
    _a[filterAndLogic] = 'And',
    _a[filterOrLogic] = 'Or',
    _a[filterAddExpression] = 'Add Expression',
    _a[filterAddGroup] = 'Add Group',
    _a[filterClose] = 'Close',
    _a);
