// @progress/kendo-react-data-tools/dist/npm/messages/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var _a;
/**
 * @hidden
 */
exports.pagerInfo = 'pager.info';
/**
 * @hidden
 */
exports.pagerFirstPage = 'pager.firstPage';
/**
 * @hidden
 */
exports.pagerPreviousPage = 'pager.previousPage';
/**
 * @hidden
 */
exports.pagerNextPage = 'pager.nextPage';
/**
 * @hidden
 */
exports.pagerLastPage = 'pager.lastPage';
/**
 * @hidden
 */
exports.pagerItemPerPage = 'pager.itemsPerPage';
/**
 * @hidden
 */
exports.pagerPage = 'pager.page';
/**
 * @hidden
 */
exports.pagerOf = 'pager.of';
/**
 * @hidden
 */
exports.pagerTotalPages = 'pager.totalPages';
/**
 * @hidden
 */
exports.filterEqOperator = 'filter.eqOperator';
/**
 * @hidden
 */
exports.filterNotEqOperator = 'filter.notEqOperator';
/**
 * @hidden
 */
exports.filterIsNullOperator = 'filter.isNullOperator';
/**
 * @hidden
 */
exports.filterIsNotNullOperator = 'filter.isNotNullOperator';
/**
 * @hidden
 */
exports.filterIsEmptyOperator = 'filter.isEmptyOperator';
/**
 * @hidden
 */
exports.filterIsNotEmptyOperator = 'filter.isNotEmptyOperator';
/**
 * @hidden
 */
exports.filterStartsWithOperator = 'filter.startsWithOperator';
/**
 * @hidden
 */
exports.filterContainsOperator = 'filter.containsOperator';
/**
 * @hidden
 */
exports.filterNotContainsOperator = 'filter.notContainsOperator';
/**
 * @hidden
 */
exports.filterEndsWithOperator = 'filter.endsWithOperator';
/**
 * @hidden
 */
exports.filterGteOperator = 'filter.gteOperator';
/**
 * @hidden
 */
exports.filterGtOperator = 'filter.gtOperator';
/**
 * @hidden
 */
exports.filterLteOperator = 'filter.lteOperator';
/**
 * @hidden
 */
exports.filterLtOperator = 'filter.ltOperator';
/**
 * @hidden
 */
exports.filterIsTrue = 'filter.isTrue';
/**
 * @hidden
 */
exports.filterIsFalse = 'filter.isFalse';
/**
 * @hidden
 */
exports.filterAfterOrEqualOperator = 'filter.afterOrEqualOperator';
/**
 * @hidden
 */
exports.filterAfterOperator = 'filter.afterOperator';
/**
 * @hidden
 */
exports.filterBeforeOperator = 'filter.beforeOperator';
/**
 * @hidden
 */
exports.filterBeforeOrEqualOperator = 'filter.beforeOrEqualOperator';
/**
 * @hidden
 */
exports.filterAndLogic = 'filter.andLogic';
/**
 * @hidden
 */
exports.filterOrLogic = 'filter.orLogic';
/**
 * @hidden
 */
exports.filterAddExpression = 'filter.addExpression';
/**
 * @hidden
 */
exports.filterAddGroup = 'filter.addGroup';
/**
 * @hidden
 */
exports.filterClose = 'filter.close';
/**
 * @hidden
 */
exports.messages = (_a = {},
    _a[exports.pagerItemPerPage] = 'items per page',
    _a[exports.pagerInfo] = '{0} - {1} of {2} items',
    _a[exports.pagerFirstPage] = 'Go to the first page',
    _a[exports.pagerPreviousPage] = 'Go to the previous page',
    _a[exports.pagerNextPage] = 'Go to the next page',
    _a[exports.pagerLastPage] = 'Go to the last page',
    _a[exports.pagerPage] = 'Page',
    _a[exports.pagerOf] = 'Of',
    _a[exports.pagerTotalPages] = '{0}',
    _a[exports.filterEqOperator] = 'Is equal to',
    _a[exports.filterNotEqOperator] = 'Is not equal to',
    _a[exports.filterIsNullOperator] = 'Is null',
    _a[exports.filterIsNotNullOperator] = 'Is not null',
    _a[exports.filterIsEmptyOperator] = 'Is empty',
    _a[exports.filterIsNotEmptyOperator] = 'Is not empty',
    _a[exports.filterStartsWithOperator] = 'Starts with',
    _a[exports.filterContainsOperator] = 'Contains',
    _a[exports.filterNotContainsOperator] = 'Does not contain',
    _a[exports.filterEndsWithOperator] = 'Ends with',
    _a[exports.filterGteOperator] = 'Is greater than or equal to',
    _a[exports.filterGtOperator] = 'Is greater than',
    _a[exports.filterLteOperator] = 'Is less than or equal to',
    _a[exports.filterLtOperator] = 'Is less than',
    _a[exports.filterIsTrue] = 'Is true',
    _a[exports.filterIsFalse] = 'Is false',
    _a[exports.filterAfterOrEqualOperator] = 'Is after or equal to',
    _a[exports.filterAfterOperator] = 'Is after',
    _a[exports.filterBeforeOperator] = 'Is before',
    _a[exports.filterBeforeOrEqualOperator] = 'Is before or equal to',
    _a[exports.filterAndLogic] = 'And',
    _a[exports.filterOrLogic] = 'Or',
    _a[exports.filterAddExpression] = 'Add Expression',
    _a[exports.filterAddGroup] = 'Add Group',
    _a[exports.filterClose] = 'Close',
    _a);
